hphp/runtime/vm/jit/vasm-ppc64.cpp

```diff
@@ -707,6 +712,14 @@ void Vgen::emit(const mcprep& i) {
 }
 
 void Vgen::emit(const inittc&) {
```

Comment:
Actually this VASM is handling the frame fix in the `emitEnterTCHelper`, which is architecture independent. Any suggestion if we must create a new VASM or any other way to do that? Also I had to change the place it is called, to be the first executed when entering the VM.